vmdb/app/models/brand.rb

```diff
@@ -0,0 +1,62 @@
+class Brand
+  attr_accessor :customer_name
+  attr_accessor :vmdb_name
+  attr_accessor :custom_logo
+
+  def initialize(attrs = {})
+    attrs.each do |n, v|
+      public_send("#{n}=", v)
+    end
+  end
+
+  def exist?
+    customer_name && vmdb_name && custom_logo
+  end
+
+  def class_text
+    custom_logo ? "whitelabeled" : ""
+  end
```

Comment:
This doesn't feel very modely...seems more UI specific...maybe it should be a presenter? cc @martinpovolny